generate_sbom

```diff
@@ -572,7 +578,45 @@ while (@ARGV && $ARGV[0] =~ /^-/) {
 $format ||= 'spdx';
 die("unknown format $format\n") unless $format eq 'spdx' || $format eq 'cyclonedx';
 
-die("usage: generate_sbom [--distro NAME] [--format spdx|cyclonedx] [--intoto] [--product DIRECTORY]|[--rpmmd DIRECTORY|CONTAINER_TAR]\n") unless @ARGV == 1;
+sub echo_help {
+    print "\n
+The Software Bill of Materials (SBOM) generation tool
+=====================================================
+
+This tool generates SBOM data based on data from rpm packages.
+
+Output formats
+==============
+
+  --format spdx
+     Generates SPDX 2.3 formated data. This is the default.
+
+  --format cyclonedx
+     Generates CycloneDX 1.4 formated data
+
+  --intoto
+     Can be used optional to wrap the generated data into in-toto.io
+     specified format.
+
+Supported content
+=================
+
+  --dir DIRECTORY
+     A Linux installation to scan. This can be any directory structure
+     which provides an installed system.
```

Comment:
The RPM database of the system below DIRECTORY will be evaluated, also all files will be referenced in the SBOM.

Reply:
changed